src/path.cpp

```diff
@@ -41,13 +45,26 @@ Path::Path(EIP_USINT class_id, EIP_USINT instance_id, EIP_USINT attribute_id,
   addLogicalClass(class_id);
   addLogicalInstance(instance_id);
   addLogicalAttribute(attribute_id);
+  //this->print();
```

Comment:
Please remove (here and elsewhere).

Reply:
yes, they can be removed. They were used for debugging.